.packit.yml

```diff
@@ -19,10 +19,10 @@ jobs:
         - fedora-all
         - epel-7-x86_64
         - epel-8-x86_64
-
-  - <<: *copr
-    trigger: commit
-    metadata:
+        - epel-9-x86_64
       owner: "@rpm-software-management"
```

Comment:
Seems like you are moving the pull-request builds into the production project, not sure?

Reply:
Oh, you are right, that's wrong. But we need to specify the same targets for the `trigger: commit` though.

Reply:
Because this was submitted for the last commit in the `main` branch https://github.com/rpm-software-management/tito/runs/33928767738 and shouldn't have been.